web-ui/src/pages/CheckinsPage.test.js

```diff
@@ -1,11 +1,3 @@
 import React from "react";
 import CheckinsPage from "./CheckinsPage";
 import { AppContextProvider } from "../context/AppContext";
-
-it("renders correctly", () => {
-    <AppContextProvider>
-        snapshot(
-        <CheckinsPage />
-        );
-    </AppContextProvider>;
-});
```

Comment:
How'd this get in here?

Reply:
Sonar was complaining about it and I was hoping to force the workflow to succeed by commenting it out. Then I forgot to undo it before I pushed.